cube/cube.c

```diff
@@ -4143,7 +4143,10 @@ static void demo_init_vk(struct demo *demo) {
     }
 
     volkLoadInstance(demo->inst);
+}
 
+static void demo_select_physical_device(struct demo* demo) {
```

Comment:
Seperate physical device to a function because it needs create surface between creating instance and selecting physical device.